### PR TITLE
PULL_TO_FINALLY_SOLVE_ISSUE#9

### PR DESCRIPTION
**Pull request by caemadmin**
/solves #9

### CODE REVIEW

File: file-issue-nine.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('issue nine');
```
Comment (caemadmin) [inline file-issue-nine.js:1]: review comment 1

**Review by caemadmin — COMMENTED** [review]
This is the firest review

**Comment by caemadmin** [comment]
Pull comment on PR#20

**Comment by caemadmin** [comment]
Comment #3 on pull request #20

File: file-issue-nine.js (hunk 1 above)
Comment (cmedwind) [reply file-issue-nine.js:1]: review_comment2_by_developer

Comment (cmedwind) [reply file-issue-nine.js:1]: hello I am developer

File: file-issue-nine.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+console.log('test 1');
```
Comment (cmedwind) [inline file-issue-nine.js:4]: This is correct

Comment (cmedwind) [reply file-issue-nine.js:4]: Review_comment_2

Comment (cmedwind) [reply file-issue-nine.js:4]: review_comment_number3

**Review by cmedwind — APPROVED** [review]

File: file-issue-nine.js (hunk 1 above)
Comment (caemadmin) [reply file-issue-nine.js:1]: think_this_is_a pull_request_review_comment.created

Comment (caemadmin) [reply file-issue-nine.js:1]: comment for event pull_request_review_comment.created

Comment (caemadmin) [reply file-issue-nine.js:1]: New PULL_REVIEW comment

File: file-issue-nine.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 2');
+console.log('test 7');
+console.log('test 9');
```
Comment (caemadmin) [inline file-issue-nine.js:7]: PRREVIEV#288 COMMENT#1

File: file-issue-nine.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+console.log('test 6');
```
Comment (caemadmin) [inline file-issue-nine.js:8]: PRREVIEV#288 COMMENT#2 hello

**Pull request closed without merging**